poker/utils/chip.py

```diff
@@ -1,54 +1,53 @@
-from collections import defaultdict
+from collections import Counter
 
-from poker.utils.constants import INCREMENT_LIMIT, SINGLE_CHIP, DOUBLE, Chip, Cash
+from poker.utils.constants import INCREMENT_LIMIT, SINGLE_CHIP, Cash
 from poker.utils.exception import IncrementException, CashException
 
 
-class Stack:
+class PlayerStack:
+
+    WHITE = {"name": "White", "value": 1}
+    RED = {"name": "Red", "value": 5}
+    BLUE = {"name": "Blue", "value": 10}
 
     def __init__(self) -> None:
-        self.chips = defaultdict(int)
+        self.chips = Counter()
     
-    # increment chips
     def increment(self, cash: int) -> None:
         if cash > INCREMENT_LIMIT:
-            raise IncrementException()
+            raise IncrementException
         
         chip_values = [item.value for item in Cash]
         if cash not in chip_values:
-            raise CashException()
+            raise CashException
         
         if cash == Cash.FIVE.value:
-            self.chips[Chip.WHITE.name] += SINGLE_CHIP * Cash.FIVE.value
+            self.chips[self.WHITE["name"]] += SINGLE_CHIP * Cash.FIVE.value
         elif cash == Cash.TEN.value:
-            self.chips[Chip.WHITE.name] += SINGLE_CHIP * Cash.FIVE.value
-            self.chips[Chip.RED.name] += SINGLE_CHIP
+            self.chips[self.WHITE["name"]] += SINGLE_CHIP * Cash.TEN.value
         elif cash == Cash.FIFTEEN.value:
-            self.chips[Chip.RED.name] += SINGLE_CHIP
-            self.chips[Chip.BLUE.name] += SINGLE_CHIP
+            self.chips[self.WHITE["name"]] += SINGLE_CHIP * Cash.FIFTEEN.value
```

Comment:
If it's a self.attr then it's not a constant right? So better lowercase them?

Reply:
Thanks for the guidance here.